test/channels/test_run_channel_test.rb

```diff
@@ -0,0 +1,16 @@
+require "test_helper"
+
+class TestRunChannelTest < ActionCable::Channel::TestCase
+  test ".broadcast! broadcasts test run" do
```

Comment:
If this is okay, I can also add tests for the `IterationChannel` and `IterationsChannel` classes.

Reply:
This looks good to me.

Reply:
I've added tests for the other two untested channels too.